backend/src/ledgers/data.ts

```diff
@@ -256,3 +257,18 @@ export async function updateCache(
   );
   await redisClient.set(pagingTokenKey, pagingToken);
 }
+
+export async function getOpStats() {
+  const query = `
+    SELECT 
+      FORMAT_DATE('%Y-%m', cast(closed_at as date)) as closing_date,
+      SUM(operation_count) as operations 
```

Comment:
let's use the tx_set_operation_count field, which gives all the operations (operation_count just gives the successful ones)
```suggestion
      SUM(tx_set_operation_count) as operations 
```

Reply:
I tested locally, it checks out!